docs/guides/testing.md

```diff
@@ -57,6 +57,11 @@ this means your application logic does not need to be changed or mocked when wri
 
 The mock provided below will enable the relevant test runner to reset the zustand stores after each test.
 
+> **Warning:** on these examples we are supporting only the curried version `create()(...)` since that
+> works for JavaScript and Typescript, if you are using the uncurried version `create(...)` you need
+> to update your hooks to use the curried version or update the mocks in order to support the uncurried
+> version.
+
```

Comment:
This is looking good. A few tweaks:

> **Warning:** In the following examples we are only demostrating the curried version of create (`create()(...)`), because that is compatible with both 
> JavaScript and Typescript. If you are using the uncurried version of `create(...)` you will need
> to update your hooks to use the curried version, or update the mocks in order to support the uncurried
> version.

Reply:
Thanks for looking through the feedback. 

I think this section "due to that is compatible" is still not quite right? 

Also "both JavaScript and Typescript, If you" is not quite right either. If you want to keep the comma, "If" will need to be "if". However, I think that sentence is very long, and it would be worth breaking it up - similar to the suggestion above. 

What are your thoughts?

Reply:
@chrisk-7777 what you think about "due to that is supported on both" instead of "due to that is compatible"

Reply:
I think the main issue is "due to that is" doesn't really read well in my opinion. 
I'm happy to a second opinion though :)

Reply:
@chrisk-7777 because of that doesn't have the same meaning of due to that, does it?

Reply:
Let me reword it